reproducible-build/build.sh

```diff
@@ -18,3 +18,8 @@ source setup || true
 make build || true
 
 cp --dereference /build/srcs/poky/build/tmp/deploy/images/tdx/* /artifacts/.
+
+make gen-measurements || true
+
+mkdir -p /artifacts/measurements
+cp --dereference /build/measurements/* /artifacts/measurements/.
```

Comment:
I understand what you did here because the `make gen-measurements` might fail due to missing packages within the docker itself. 
Have you tested if it works? because I believe the docker file needs to be accordingly updated for this to work as it might not have the right dependencies & python modules that are described above to generate the measurements